.gitmodules

```diff
@@ -42,8 +42,7 @@
 	url = https://github.com/dotty-staging/scala-xml
 [submodule "community-build/community-projects/shapeless"]
 	path = community-build/community-projects/shapeless
-	url = https://github.com/milessabin/shapeless
-	branch = shapeless-3
+	url = https://github.com/dotty-staging/shapeless
```

Comment:
@milessabin, to make some small changes to shapeless I forked it and used a new branch. I will upstream those changes once this PR is merged.